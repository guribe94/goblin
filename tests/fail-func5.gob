world[20,40] {

}
entities {
  @:player {
    build {
    }
    does {
      test(this);
      exit;
    }
  }

}
functions {

  bool foo(num a, bool b)
  {
    return true;
  }

  bool test(entity p) {
    foo(3, false);
    foo(55);
    return true;
  }
}
