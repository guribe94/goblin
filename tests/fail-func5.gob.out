Fatal error: exception Failure("expecting 2 arguments in foo(55)")
